src/MAUI/Maui.Samples/Samples/Data/GeodatabaseTransactions/GeodatabaseTransactions.xaml.cs

```diff
@@ -105,53 +109,53 @@ private async Task GetLocalGeodatabase()
                             {
-                                // Hide the progress control and report the exception
+                                // Hide the progress control and report the exception.
                                 LoadingProgressBar.IsVisible = false;
                                 MessageTextBlock.Text = "Unable to create local geodatabase: " + generateGdbJob.Error.Message;
                             });
                         }
                     };
 
-                    // Start the generate geodatabase job
+                    // Start the generate geodatabase job.
                     _localGeodatabase = await generateGdbJob.GetResultAsync();
                 }
             }
             catch (Exception ex)
             {
-                // Show a message for the exception encountered
+                // Show a message for the exception encountered.
                 Microsoft.Maui.ApplicationModel.MainThread.BeginInvokeOnMainThread(() =>
                 {
                     Application.Current.MainPage.DisplayAlert("Generate Geodatabase", "Unable to create offline database: " + ex.Message, "OK");
                 });
             }
         }
 
-        // Function that loads the two point tables from the local geodatabase and displays them as feature layers
+        // Function that loads the two point tables from the local geodatabase and displays them as feature layers.
         private async Task LoadLocalGeodatabaseTables()
         {
             if (_localGeodatabase == null) { return; }
 
-            // Read the geodatabase tables and add them as layers
+            // Read the geodatabase tables and add them as layers.
             foreach (GeodatabaseFeatureTable table in _localGeodatabase.GeodatabaseFeatureTables)
             {
                 try
                 {
-                    // Load the table so the TableName can be read
+                    // Load the table so the TableName can be read.
                     await table.LoadAsync();
 
-                    // Store a reference to the Birds table
+                    // Store a reference to the Birds table.
                     if (table.TableName.ToLower().Contains("birds"))
                     {
                         _birdTable = table;
                     }
 
-                    // Store a reference to the Marine table
+                    // Store a reference to the Marine table.
                     if (table.TableName.ToLower().Contains("marine"))
                     {
                         _marineTable = table;
                     }
 
-                    // Create a new feature layer to show the table in the map
+                    // Create a new feature layer to show the table in the map.
                     FeatureLayer layer = new FeatureLayer(table);
                     Microsoft.Maui.ApplicationModel.MainThread.BeginInvokeOnMainThread(() => MyMapView.Map.OperationalLayers.Add(layer));
```

Comment:
On iOS sim, this line of code is throwing a `NullReferenceException`:
![image](https://github.com/Esri/arcgis-maps-sdk-dotnet-samples/assets/118313312/8c404447-e8bf-4acf-a857-0c9cc442b33a)
I can also repro this on v.next.

Reply:
So this fails even without this PR changes? Are you able to check which property is null on this line? Is it MapView, Map or ApplicationModel or MainThread?

Reply:
Yep, this issue pre-dates this PR. To repro, launch sample and return to category page. MapView.Map is set as null [OnNavigatedFrom](https://github.com/Esri/arcgis-maps-sdk-dotnet-samples/blob/v.next/src/MAUI/Maui.Samples/SamplePage.xaml.cs#L114) so before my breakpoint on line 160 is hit. I will see if I can repro on other MAUI platforms.

Reply:
If samples is setting Map to null, then we need a null check here. May be a different PR, sounds like it can happen else where Map is accessed on an invoke after you navigated away from a sample.

Reply:
Can confirm using null operator (MyMapView.Map?.OperationalLayers) on line 160 prevents this exception on iOS.